src/ESPiLight.h

```diff
@@ -71,6 +71,11 @@ class ESPiLight {
   void setCallback(ESPiLightCallBack callback);
   void setPulseTrainCallBack(PulseTrainCallBack rawCallback);
 
+  /**
+   * If set to true, the receiver will temporarely be disabled when sending.
+   */
+  void setEchoEnabled(bool enabled);
```

Comment:
[Arduino Style Guide for Writing Libraries](https://www.arduino.cc/en/Reference/APIStyleGuide) says:
> Try to avoid boolean arguments. Instead, consider providing two different functions with names the describe the differences between them.

How about: `enableEcho()` and `diableEcho()`

Reply:
Well, I use this from a config-vaiable that is set to true or false. So for me a setter would be more convinient. I think thats the main reason why a boolean argument might be OK here: Its a setter.

Reply:
I see that it is more convenient for you, but the Style guide is clear in this case.

Reply:
I think what the style guide means here is that you should not use boolean flags to influence the behaviour of a method. For example instead of 
```
void connect(const String& host, const int port, const bool use_udp) {
... 
  if (use_udp) {
  ...
  }
}
```
write
```
void connectTcp(const String& host, const int port) {
  ...
}
void connectUdp(const String& host, const int port) {
  ...
}
```
Which comes from "Robert C. Martin's Clean Code Tip #12: Eliminate Boolean Arguments" wwich states, that a boolean flag is a sign that a function does more than one thing. As we actually do nothing here than setting a configuration value I would say its better that way.

Reply:
Did you have any example of an popular lib that use boolean setter?

Reply:
As I just had the code open:
* https://github.com/tzapu/WiFiManager/blob/master/WiFiManager.h#L111
* https://github.com/tzapu/WiFiManager/blob/master/WiFiManager.h#L91
* https://github.com/esp8266/Arduino/blob/master/libraries/ESP8266WiFi/src/ESP8266WiFiGeneric.h#L77
* https://github.com/esp8266/Arduino/blob/master/libraries/ESP8266WiFi/src/ESP8266WiFiGeneric.h#L82-#L83
* https://github.com/esp8266/Arduino/blob/master/libraries/ESP8266WiFi/src/ESP8266WiFiSTA.h#L52

Reply:
ok